File: DailyFlash Solutions/DailyFlash Solutions/60-DailyFlash_Solutions/30_Mar_Solution_One/CPP/Program0.cpp
#include <stdio.h>

float factorial(int num){

	float fact = 1;
	for(int i = 1; i <= num; i++){

		fact*=i;
	}
	return fact;
}

int main() {

	int l = 0;
	float sum = 0;
	printf("Enter The Length of the Series\n");
	scanf("%d",&l);

	for(int i = 1; i <= l; i++){

		sum+=(1/factorial(i));
	}

	printf("Sum of Series = %.3f\n",sum);
	return 0;
}

File: DailyFlash Solutions/DailyFlash Solutions/60-DailyFlash_Solutions/30_Mar_Solution_One/Java/Program0.java
import java.io.*;



class Program0 {


	static float factorial(int num){

		float fact = 1;
		for(int i = 1; i <= num; i++){

			fact*=i;
		}
		return fact;
	}

	public static void main(String[] args) throws IOException {

		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
		float sum = 0;
		int l = 0;
		System.out.printf("Enter The Length of the Series\n");
		l = Integer.parseInt(br.readLine());

		for(int i = 1; i <= l; i++)
			sum+=(1/Program0.factorial(i));
		System.out.println("Sum of Series = "+String.format("%.4f",(float)sum));
	}
}
